js/datagrid.ts

```diff
@@ -189,10 +189,11 @@ export class DataGridModel extends DOMWidgetModel {
 
         this.synchingWithKernel = true;
 
-        const selectionIter = sender.selections().iter();
+        const selectionIter = sender.selections();
         const selections: any[] = [];
-        let selection = null;
-        while ((selection = selectionIter.next())) {
+        let selectionNode = null;
+        while ((selectionNode = selectionIter.next())) {
+          const selection = selectionNode.value;
```

Comment:
This fixes https://github.com/bloomberg/ipydatagrid/issues/431

Reply:
But is backwards incompatible with older versions of lumino which I personally think is fine but probably means we should bump the minor version.

Reply:
Let's try making something that works with both. I will look into it.

Reply:
Done!